core/templates/tiddlywiki5.html.tid

```diff
@@ -41,6 +41,7 @@ title: $:/core/templates/tiddlywiki5.html
 </noscript>
 <!--~~ Ordinary tiddlers ~~-->
 `{{$:/core/templates/store.area.template.html}}`
+`{{$:/core/templates/store.info.template.html}}`
```

Comment:
@Jermolene -- I would like to convert this PR into a plugin.

The absolute minimum required in the core would be this one line. So the whole mechanism would be similar to ` $:/tags/RawMarkup*` behaviour. 

To make it less error prone we should also keep the template `$:/core/templates/store.info.template.html` tiddler in the core, which defines how to handle "injected" tiddlers. 

What do you think?

Reply:
Hi @pmario that makes sense. Rather than adding another hardcoded template title we could use a filtered transclusion to bring in any tiddlers with the appropriate tag?

Reply:
IMO that's exactly what: https://github.com/Jermolene/TiddlyWiki5/blob/7b0f23b2cc05c0fead63ec7265a5d54d940b2d52/core/templates/store.info.template.html.tid does.

Reply:
Hi @pmario if it was a generic mechanism for splicing tiddlers into saved wikis then it probably shouldn't have the name "info" in the title.

Reply:
It basically is an independent "store-area" to minimise side effects, if users mess it up. So may be we could use `$:/core/templates/custom.store.template.html` or  `$:/core/templates/store.custom.template.html` if we want it to start with `/store.`
 
The tag at the moment is: `$:/tags/InfoTiddlerTemplate` which also would need to be renamed eg: `$:/tags/CustomStore`

So the code in the core template would look like this:

```
<!--~~ Ordinary tiddlers ~~-->
`{{$:/core/templates/store.area.template.html}}`
`{{$:/core/templates/custom.store.template.html}}`
```

Reply:
Hi @pmario I did notice the use of a separate store area. I think that would mean that these tiddlers wouldn't be visible to the import process?

Reply:
I'm not sure, what you mean with "import process". 

When the wiki is reloaded, they will be "merged" into the main wiki-store as standard tiddlers.

Reply:
I meant that an attempt to import the saved wiki (eg via drag and drop) would not see the inserted tiddlers.

Reply:
OK - I did not think about that. I'll need to test this.

Reply:
- I did download the [preview wiki](https://tiddlywiki5-git-fork-pmario-6684-add-last-save-16b5df-jermolene.vercel.app/)
- I did save empty from the preview
- Open empty.html and -> save it so new tiddlers are injected
- Did open empty.html in **text editor**
- Main store: `last-saved/date: 2024-03-10 11:11:42.171 UTC +01:00` see: 11:11
- Custom store exists with `last-saved/date: 2024-03-10 11:16:31.158 UTC +01:00` see: 11:16
- Import into tiddlywiki.com -> **seems to work** as expected. 
- ![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/3472ed80-07a2-4285-8c7d-900a8a4a76b1)

Reply:
Thanks @pmario I had a look at the code in `$:/core/modules/deserializers.js` and it does indeed process multiple new style store areas.

Reply:
So -- which "generic" names would you suggest?